src/OpenSage.Tools.BigEditor/UI/MainForm.cs

```diff
@@ -59,14 +66,18 @@ private void UpdateSearch(string searchText)
 
         public void Draw(Sdl2Window window)
         {
+            _window = window;
```

Comment:
Why you store the window as a member? I don't see it being used somewhere

Reply:
Okay, i found it but better pass the width & height to `DrawFileContent`, so we don't need that member

Reply:
It used for getting width and height of window in the DrawFilesList function at 355 line

Reply:
Yes, better pass a `Vector2` to DrawFilesList with the size. Because the window is only valid during the "scope" of the Draw function. So:
```csharp
 private void DrawFilesList(Vector2 windowSize)
{
.
.
  ImGui.SetWindowSize(windowSize - 50, ImGuiCond.Always);
.
.
}
```

Reply:
Is it possible subtract int from Vector2 ?
it;s different types

Reply:
You need to do `- new Vector2(50)`